cluster/src/main/scala/org/apache/spark/sql/SnappyContextFactory.scala

```diff
@@ -78,9 +78,9 @@ trait SnappySQLJob extends SparkJobBase {
     }
   }
 
-  def isValidJob(sc: SnappyContext, config: Config): SnappyJobValidation
+  def isValidJob(sc: SnappySession, config: Config): SnappyJobValidation
 
-  def runSnappyJob(sc: SnappyContext, jobConfig: Config): Any
+  def runSnappyJob(sc: SnappySession, jobConfig: Config): Any
 
 }
```

Comment:
Should this class be called SnappyContextFactory anymore or SnappySessionFactory?